src/CSharp/CodeCracker/Style/RemoveAtFromVariablesAnalyzer.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace CodeCracker.CSharp.Style
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class RemoveAtFromVariablesAnalyzer : DiagnosticAnalyzer
+    {
+        internal const string Title = "Remove @ from variables that are not keywords.";
+        internal const string MessageFormat = "Remove @ from variables that are not keywords.";
+        internal const string Category = SupportedCategories.Style;
+        const string Description = "Usage of @ on variable names only when it is a CSharp keyword";
```

Comment:
@giggio Localizable Strings is something new to Code Craker Analyzers? Most of them are like this one that I've created...